bin/replication.js

```diff
@@ -115,8 +114,10 @@ class _SetupReplication {
             'default' : sourceProfile;
         const verifyTargetProfile = targetProfile === undefined ?
             'default' : targetProfile;
-        source.auth.vault.host = 'localhost';
-        source.s3.host = 'localhost';
+
+        // TODO: Federation fix made #1125?
+        // source.auth.vault.host = 'localhost';
+        // source.s3.host = 'localhost';
         const destHost = this.destHosts.pickHost().host;
```

Comment:
If I'm not mistaken, you should pick both `host` and `port` from the result of `pickHost`, then construct the endpoint string as `host:port`.